CIP-0137/README.md

```diff
@@ -450,7 +450,8 @@ The local message notification mini-protocol is used by local clients to be noti
 The protocol follows a simple request-response pattern:
 
 1. The client sends a request with a single message.
-2. The server either accepts the message (and returns a confirmation) or rejects it (and returns the reason)
+2. If the requests is non blocking, the server either accepts it (and returns a message and a flag stating if further messages are available) or rejects it (there is no message in that case).
+3. If the requests is blocking, the server either accepts the request (and, when availabe, returns a message and a flag stating if further messages are available) or stops the protocol.
```

Comment:
Is it really necessary to have blocking/non-blocking states here? For such a simple req/rep protocol we just want to avoid the client side to do unnecessary polling. We already know that we can expect a new message every 10 minutes so polling every 10 minutes is ideal, if the server has received more messages in that time span we can reply with how many more new messages the client can still fetch. After the client receives a 0 value or a `MsgTimeoutMessage NoNewMessages` (e.g.) it can poll the server again in 10 minutes or so.